src/main/java/org/gridsuite/study/server/service/NetworkMapService.java

```diff
@@ -49,6 +50,33 @@ public NetworkMapService(
         this.networkMapServerBaseUri = networkMapServerBaseUri;
     }
 
+    public String getElementsInfos(UUID networkUuid, String variantId, List<String> substationsIds, String elementType, String infoType) {
+        String path = DELIMITER + NETWORK_MAP_API_VERSION + "/networks/{networkUuid}/elements";
+        UriComponentsBuilder builder = UriComponentsBuilder.fromPath(path);
+        if (substationsIds != null) {
+            builder = builder.queryParam(QUERY_PARAM_SUBSTATIONS_IDS, substationsIds);
+        }
+        if (!StringUtils.isBlank(variantId)) {
+            builder = builder.queryParam(QUERY_PARAM_VARIANT_ID, variantId);
+        }
+        builder = builder.queryParam(QUERY_PARAM_ELEMENT_TYPE, elementType);
+        builder = builder.queryParam(QUERY_PARAM_INFO_TYPE, infoType);
+        String url = builder.buildAndExpand(networkUuid).toUriString();
+        return restTemplate.getForObject(networkMapServerBaseUri + url, String.class);
+    }
+
+    public String getElementInfos(UUID networkUuid, String variantId, String elementType, String infoType, String elementId) {
+        String path = DELIMITER + NETWORK_MAP_API_VERSION + "/networks/{networkUuid}/elements/{elementId}";
+        UriComponentsBuilder builder = UriComponentsBuilder.fromPath(path);
+        if (!StringUtils.isBlank(variantId)) {
+            builder = builder.queryParam(QUERY_PARAM_VARIANT_ID, variantId);
+        }
+        builder = builder.queryParam(QUERY_PARAM_ELEMENT_TYPE, elementType);
+        builder = builder.queryParam(QUERY_PARAM_INFO_TYPE, infoType);
+        String url = builder.buildAndExpand(networkUuid, elementId).toUriString();
+        return restTemplate.getForObject(networkMapServerBaseUri + url, String.class);
+    }
+
     public String getEquipmentsMapData(UUID networkUuid, String variantId, List<String> substationsIds,
```

Comment:
can be removed, using getElementsInfos
same for getEquipmentMapData, getEquipmentsIds

Reply:
Done in a another US